api/resources/views/vendor/mail/html/table.blade.php

```diff
@@ -0,0 +1,3 @@
+<div class="table">
```

Comment:
Do we need to export all those blade files? If there is an update, we'll need to update all manually.
Nothing is changed I guess to this file